templates/generated-readmes.old/ngc-rapidsai-clx-dev.md

```diff
@@ -0,0 +1,205 @@
+#  RAPIDS - Open GPU Data Science
```

Comment:
can you remove the `templates/generated-readmes.old/` directory and all of its contents? This shouldn't have been committed.